### PR TITLE
[WIP] housing-assistance

### PR DESCRIPTION
**Pull request by rroueche**
From this issue: https://github.com/department-of-veterans-affairs/vets.gov-content/issues/236

### CODE REVIEW

**Comment by rroueche** [comment]
@bethpotts ready for you to pour the content in!

**Comment by bethpotts** [comment]
@rroueche Thank you very kindly for stubbing these out! @DanielleSoCompany Here's the PR and branch (content/wip/housing-assistance) for uploading content.

**Comment by bethpotts** [comment]
@rroueche Wanted to confirm where we landed with the url for the apply page: apply-for-certificate-of-eligibility. Sound right? If so, I think the folder name needs to change.

**Comment by rroueche** [comment]
@bethpotts yep you are correct! apply-for-certificate-of-eligibility i'll do that now and will confirm here when it's ready

File: content/pages/housing-assistance/adaptive-housing-grants.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+</div>
+
+<div class="feature" markdown=“1”>
+
+### Can I get a Specially Adapted Housing (SAH) grant—and how much funding does this grant offer? 
+
+You may be able to get an SAH grant if you’re using the grant money to buy, build, or change your permanent home (a home you plan to live in for a long time) if:
+
+-	You own or will own the home, **and**
```
Comment (melwoodard) [inline content/pages/housing-assistance/adaptive-housing-grants.md]: @bethpotts this is one of those formatting errors Rachael mentioned from copy/pasting from Word - even though when you edit the file the bullet and spacing looks correct, you can see here in the code output that there's a bunch of extra space between the bullet and the text. That unfortunately causes the sub-bullets to not indent correctly.

I'll fix this here, but hopefully this helps with troubleshooting - when things are looking wonky, try deleting and retyping the bullets or numbers in a list.

**Comment by rroueche** [comment]
@melwoodard @amybehrens @bethpotts is this good to go?

**Review by bethpotts — APPROVED** [review]
SME feedback incorporated. Proof complete. Ship!

**Comment by rroueche** [comment]
@amybehrens @bethpotts @DanielleSoCompany merging now!

**Pull request merged**